meta/.phpstorm.meta.php

```diff
@@ -136,6 +136,25 @@ function argumentsSet($setName) {
 	expectedArguments(\fopen(), 1, 'r', 'r+', 'w', 'w+', 'a', 'a+', 'x', 'x+', 'c', 'c+', 'e');
 	expectedArguments(\popen(), 1, 'r', 'r+', 'w', 'w+', 'a', 'a+', 'x', 'x+', 'c', 'c+', 'e');
 	expectedArguments(\SplFileInfo::openFile(), 0, 'r', 'r+', 'w', 'w+', 'a', 'a+', 'x', 'x+', 'c', 'c+', 'e');
+	
+	expectedArguments(\fopen(), 0, 'php://stdout', 'php://stderr', 'php://stdin', 'php://input', 'php://output', 'php://fd/', 'php://memory', 'php://temp', 'php://temp/maxmemory=', 'php://filter/resource=', 'php://filter/read=', 'php://filter/write=', 'file://', 'http://', 'ftp://', 'ftps://', 'zip://', 'compress.zlib://', 'compress.bzip2://', 'data://text/plain,', 'data://text/plain;base64,', 'phar://', 'ssh2.shell://', 'ssh2.exec://', 'ssh2.tunnel://', 'ssh2.sftp://',  'ssh2.scp://', 'rar://', 'ogg://', 'expect://');
+	expectedArguments(\readfile(), 0, 'php://stdin', 'php://input', 'php://output', 'php://fd/', 'php://temp', 'php://temp/maxmemory=', 'php://filter/resource=', 'php://filter/read=', 'file://', 'http://', 'ftp://', 'ftps://', 'zip://', 'compress.zlib://', 'compress.bzip2://', 'data://text/plain,', 'data://text/plain;base64,', 'phar://', 'ssh2.shell://', 'ssh2.exec://', 'ssh2.tunnel://', 'ssh2.sftp://', 'ssh2.scp://', 'rar://', 'ogg://', 'expect://');
+	expectedArguments(\file(), 0, 'php://stdin', 'php://input', 'php://output', 'php://fd/', 'php://temp', 'php://temp/maxmemory=', 'php://filter/resource=', 'php://filter/read=', 'file://', 'http://', 'ftp://', 'ftps://', 'zip://', 'compress.zlib://', 'compress.bzip2://', 'data://text/plain,', 'data://text/plain;base64,', 'phar://', 'ssh2.shell://', 'ssh2.exec://', 'ssh2.tunnel://', 'ssh2.sftp://', 'ssh2.scp://', 'rar://', 'ogg://', 'expect://');
+	expectedArguments(\file_get_contents(), 0, 'php://stdin', 'php://input', 'php://fd/', 'php://temp', 'php://temp/maxmemory=', 'php://filter/resource=', 'php://filter/read=', 'file://', 'http://', 'ftp://', 'ftps://', 'zip://', 'compress.zlib://', 'compress.bzip2://', 'data://text/plain,', 'data://text/plain;base64,', 'phar://', 'ssh2.shell://', 'ssh2.exec://', 'ssh2.tunnel://', 'ssh2.sftp://', 'ssh2.scp://', 'rar://', 'ogg://', 'expect://');
+	expectedArguments(\file_put_contents(), 0, 'php://stdout', 'php://stderr', 'php://output', 'php://fd/', 'php://temp', 'php://temp/maxmemory=', 'php://filter/resource=', 'php://filter/write=', 'file://', 'ftp://', 'ftps://', 'compress.zlib://', 'compress.bzip2://', 'phar://', 'ssh2.shell://', 'ssh2.exec://', 'ssh2.tunnel://', 'ssh2.sftp://', 'ogg://', 'expect://');
+	expectedArguments(\DirectoryIterator::__construct(), 0, 'glob://', 'rar://', 'phar://');
+	expectedArguments(\RecursiveDirectoryIterator::__construct(), 0, 'glob://', 'rar://', 'phar://');
+	expectedArguments(\glob(), 0, 'phar://');
+
+	expectedArguments(\fwrite(), 0, STDOUT, STDERR);
```

Comment:
Same, this line will means that basically it's not expected to see anything except `STDOUT` or `STDERR` which is not true.

Reply:
I understand. Sorry.

Reply:
I thought expectedArguments mean it is just a "weighted" suggestion and does not prevent you using other constants and scalars as well...